docs/09.EXT-REFERENCE-CONTEXT.md

```diff
@@ -0,0 +1,142 @@
+# jerryx_context types
```

Comment:
(note: I did not suggest to merge all extensions into a single doc. the question was whether to keep the numbering of all extension docs unique, or whether they should all start with 09 (but in different files). I know that it has some interworking with update-webpage.sh, but I've never used that tool.)